scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowFunctions.scala

```diff
@@ -377,14 +389,28 @@ final class SeqTFExampleSCollectionFunctions[T <: Example](private val self: SCo
    *
    * @group output
    */
+  @deprecated("saveAsTfExampleFile is deprecated: use saveAsTfRecordFile instead", "Scio 0.7.3")
```

Comment:
@regadas ... since these deprecations are new AFAIK shouldn't these be since `Scio 0.7.4` ?

Reply:
I'll fix it in an upcoming PR